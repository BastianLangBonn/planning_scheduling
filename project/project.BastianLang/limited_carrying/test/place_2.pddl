(define (problem problem1)
	(:domain domain)
	(:objects 	s - s1
			o - o1)
	(:init 	(robot_at s)
		(grapped o)
		(carries_two)	

		;Distance Metric

		(= (distance s s) 0)
		
		(= (total-cost) 0)
		
	)
	(:goal
		(and (carries_one)(not(carries_two)))
	)
)
