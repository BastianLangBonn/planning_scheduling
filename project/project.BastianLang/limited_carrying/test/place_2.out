(place_2 o s)
